types/stylelint/index.d.ts

```diff
@@ -216,21 +216,22 @@ declare module 'stylelint' {
 		_postcssResult?: PostcssResult;
 	};
 
-	export type UnusedRange = {
-		unusedRule: string;
+	export type DisableReportRange = {
+		rule: string;
 		start: number;
 		end?: number;
+
+		// This is for backwards-compatibility with formatters that were written
+		// when this name was used instead of `rule`. It should be avoided for new
+		// formatters.
+		unusedRule: string;
```

Comment:
Need to create an issue to remove it for next major version after merging PR..

Reply:
Done: https://github.com/stylelint/stylelint/issues/4899